pydrive2/fs/spec.py

```diff
@@ -167,6 +167,7 @@ def __init__(
         client_json_file_path=None,
         use_service_account=False,
         profile=None,
+        acknowledge_abuse=False,
```

Comment:
style/minor - move trash only nearby to this one

Reply:
Strictly speaking, that would be breaking backwards compatibility. But we can add an `*args` guard (which is also breaking it but at least only once).

Reply:
Introducing the guard forces changing tests, which is out of scope here. Can get back to this separately in a followup. For now moved those flags nearby, as requested.